anta/tests/routing/bgp.py

```diff
@@ -290,7 +290,7 @@ def validate_inputs(self: BaseModel) -> BaseModel:
                 """
                 Validate the inputs provided to the BgpAfi class.
 
-                If afi is either ipv4 or ipv6, safi must be provided.
+                If afi is ipv4, ipv6 or sr-te, safi must be provided.
```

Comment:
missed to update, will update back in next commit

Reply:
Yes please update this. Also, can we add a note somewhere maybe in the docstring of the test class explaining the different syntax for sr-te?

Reply:
Updated